src/components/Composable/NavbarPageCreator/NavbarPageCreator.js

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import Link from "react-router/es/Link";
+
+const NavbarPageCreator = (Component) => {
+    return () => (
+        <div>
+            <nav className="navbar navbar-default navbar-static-top" role="navigation">
+                <div className="navbar-header">
+                    <Link className="navbar-brand" to="/">Balfour</Link>
+                </div>
+                <div className="collapse navbar-collapse">
+                    <ul className="nav navbar-nav">
+                        <li><Link to="/">Dashboard</Link></li>
+                        <li><Link to="/duplicates">Duplicates</Link></li>
+                    </ul>
+                </div>
+            </nav>
+            <div className="container">
+                <Component/>
+            </div>
+        </div>
+    );
+};
+
```

Comment:
It should be reusable, not composable. You can name it NavbarPage in reusable directory and send own data as this.props.children